pkg/parse/root.go

```diff
@@ -385,9 +385,6 @@ func (p *root) setSyncStatusWithRetries(ctx context.Context, errs status.MultiEr
 
 	currentRS := rs.DeepCopy()
 
-	// syncing indicates whether the applier is syncing.
-	syncing := p.applier.Syncing()
```

Comment:
The reason I removed the Syncing call here is that the Syncing bool and Applier MultiError should come from the same time. It's already sketchy that they weren't locked, but having them be pulled at different times makes it hard to clean up. So now hopefully the next cleanup can somehow synchronize them a little better, because they're both being passed into the update function from the same source.